modules/ipi-install-firmware-requirements-for-installing-with-virtual-media.adoc

```diff
@@ -0,0 +1,35 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_bare_metal_ipi/ipi-install-prerequisites.adoc
+
+[id='ipi-install-firmware-requirements-for-installing-with-virtual-media_{context}']
+= Firmware requirements for installing with virtual media
+
+The installer for installer-provisioned {product-title} clusters validates the hardware and firmware compatibility with RedFish virtual media. The following table lists supported firmware for installer-provisioned {product-title} clusters deployed with RedFish virtual media.
+
+.Firmware Compatibility for RedFish Virtual Media
+[frame="topbot", options="header"]
+|====
+|Hardware| Model | Management | Minimum Firmware Version
+.2+| HP | 10th Generation | iLO5 | N/A
+| 9th Generation | iLO4 | N/A
+
+.2+| Dell | 14th Generation | iDRAC 9 | v4.20.20.20 - 04.40.00.00
+
+| 13th Generation .2+| iDRAC 8 | v2.75.75.75+
+
+|====
+
+[NOTE]
+====
+Refer to the hardware documentation for the nodes or contact the hardware vendor for information on updating the firmware.
+
+There are no known firmware limitations for HP servers.
+
+For Dell servers, ensure the {product-title} cluster nodes have AutoAttach Enabled through the iDRAC console. The menu path is: *Configuration* -> *Virtual Media* -> *Attach Mode* -> *AutoAttach* . With iDRAC 9 firmware version `04.40.00.00`, the Virtual Console Plugin defaults to `eHTML5`, which causes problems with the *InsertVirtualMedia* workflow. Set the Plugin to `HTML5` to avoid this issue.
```

Comment:
Maybe we should mention the path to update the Virtual Console Plugin?

Reply:
Added.